python/lsst/ap/verify/pipeline_driver.py

```diff
@@ -125,6 +138,13 @@ def runApPipe(metricsJob, workspace, parsedCmdLine):
     log = lsst.log.Log.getLogger('ap.verify.pipeline_driver.runApPipe')
 
     dataId = _parseDataId(parsedCmdLine.dataId)
+    #  After processes are implemented, remove the flake exception
+    processes = parsedCmdLine.processes  # noqa: F841
+
+    #  Insert job metadata including dataId
+    metricsJob.meta.update({'instrument': _extract_instrument_from_butler(workspace.workButler)})
+    metricsJob.meta.update(dataId)
```

Comment:
I assume this is the same tag we discussed with @afausti on DM-11321. Is it supposed to be job-level or measurement-level?

Reply:
Also, it might be worth a comment that it's `update(dataId)` because a `daf.persistence.DataId` is not serializable... just got bitten by that myself!

Reply:
It needs to be at Job level since the measurements are all ingested as a Job.  We could also record measurement level metadata, but that is out of scope for this.  As to the second comment, that is not why I used `update`.  I used it because that's the recommended way of merging dictionaries as far as I know.  If you have text for a comment, I'd be happy to add it, but I'm not sure I could come up with something on my own.

Reply:
I'm not used to thinking of either `metricsJob.meta` or a data ID as a dictionary, but ok. As long as we're all on the same page about how and where the information should be provided.